python/kfserving/README.md

```diff
@@ -45,6 +45,11 @@ KFServing supports the following storage providers:
     * Absolute path: `/absolute/path` or `file:///absolute/path`
     * Relative path: `relative/path` or `file://relative/path`
     * For local filesystem, we recommended to use relative path without any prefix.
+* Persistent Volume Claims with the format "pvc://{$pvcname}/[path]".
+    * The `pvcname` is the name of Persistent Volume Claims that storages models.
+    * The `[path]` is the relative path on the Persistent Volume Claims.
+    * For e.g. `pvc://mypvcname/model/path/on/pvc`
```

Comment:
would be great to add an example that actually showed how to use this... ie
1. have a sample PV, PVC
2. write a model to the PVC
3. use the model by referencing the PVC

Reply:
Done, please review, thanks.